src/components/companies/detail/CustomTooltip.tsx

```diff
@@ -36,12 +38,17 @@ export const CustomTooltip = ({
       };
       payload = [companyTotal, ...payload];
     }
-    
+
     const isBaseYear = companyBaseYear === payload[0].payload.year;
-    
+
     return (
-      <div className="bg-black-1 px-4 py-3 rounded-level-2 max-w-[525px] ">
-        <div className="text-sm font-medium mb-2">{label}{isBaseYear ? '*' : ''}</div>
+      <div
+        className={`${isMobile ? "w-[280px]" : "w-[400px]"} bg-black-1 px-4 py-3 rounded-level-2 `}
```

Comment:
```suggestion
        className={`${isMobile ? "max-w-[280px]" : "max-w-[400px]"} bg-black-1 px-4 py-3 rounded-level-2 `}
```
By setting these as max widths, you prevent the tooltip to be "unnecessarly" wide on desktop.

Reply:
Ahh ofcourse! Thanks for the suggestion and feedback!